cfg/Parameters.cfg

```diff
@@ -40,9 +40,9 @@ PACKAGE='control_toolbox'
 
 def generate(gen):
 	#        Name            Type    Level  Description      Default  Min  Max
-	gen.add( "p" ,      double_t, 1,"Proportional gain.", 10.0 , 0 , 100000)
-	gen.add( "i" ,      double_t, 1,"Integral gain.",     0.1 , 0 , 1000)
-	gen.add( "d" ,      double_t, 1,"Derivative gain.",   1.0 , 0 , 1000)
+	gen.add( "p" ,      double_t, 1,"Proportional gain.", 10.0 , -100000 , 100000)
```

Comment:
I have to admit I also did a bit of homework on this before putting the +1 :)

Allow me to rephrase the situation bluntly. The PID class lets you set negative gains for all, so it is best to keep this consistent with the behaviour of the `dynamic_reconfigure` window. This PR itself should be that simple.

It does however remind us that there is no test on negative gains in the class, and question is if it makes sense to implement some for that later. (Not the scope of this PR)